src/structures/CommandContext.ts

```diff
@@ -137,8 +143,13 @@ export class CommandContext {
         return this.context instanceof ButtonInteraction;
     }
 
-    public isSelectMenu(): boolean {
-        return this.context instanceof SelectMenuInteraction;
+    public isStringSelectMenu(): boolean {
+        /* NOT SURE WITH THIS CHANGES, 
+        OLD isSelectMenu method is now deprecated
+        NEW use isStringSelectMenu method  
+        ref :- https://discord.js.org/#/docs/discord.js/main/class/CommandInteraction?scrollTo=isSelectMenu
+        */
```

Comment:
<img src="https://media.tenor.com/XPzoXNYIuy8AAAAC/oozora-subaru-hololive.gif" width="200"/>

You can remove the comment.